apps/volontulo/frontend/scss/customization/_customization.scss

```diff
@@ -360,9 +360,15 @@ input.form-control[required] {
         }
     }
     .user-photo {
-        img {
-            width: 100%;
-            max-height: 300px;
+        .cover-image {
+            height: 250px;
+            background-repeat: no-repeat;
+            background-size: cover;
+            #remove-photo {
+                position: absolute;
+                top: 82%;
+                right: 10%;
```

Comment:
this looks somewhat random d:, but I'd have to switch to this branch to really find out. Maybe we save the css world in a different ticket